ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -59,16 +59,29 @@ The following steps prepare the hub cluster for site deployment and initiate ZTP
 - Push your changes to the git repository. The SiteConfig and PolicyGenTemplate CRs may be pushed simultaneously.
 
 ### Monitoring progress
-The ArgoCD pipeline uses the SiteConfig and PolicyGenTemplate CRs in GIT to generate the cluster configuration CRs & ACM policies then sync them to the hub. The progress of this synchronization can be monitored in the ArogCD dashboard.
+The ArgoCD pipeline uses the SiteConfig and PolicyGenTemplate CRs in GIT to generate the cluster configuration CRs & ACM policies then sync them to the hub.
 
-The progress of cluster installation can be monitored from the command line:  
+The progress of this synchronization can be monitored in the ArogCD dashboard.
+
+Once the synchonization is complete, the installation generally proceeds in two phases:
+
+1. The Assisted Service Operator installs OpenShift on the cluster
+
+The progress of cluster installation can be monitored from the ACM dash board, or the command line:
 ```
      $ export CLUSTER=<clusterName>
      $ oc get agentclusterinstall -n $CLUSTER $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Completed")]}' | jq
      $ curl -sk $(oc get agentclusterinstall -n $CLUSTER $CLUSTER -o jsonpath='{.status.debugInfo.eventsURL}')  | jq '.[-2,-1]'
 ```
+
+2. The Topology Aware Lifecycle Operator then applies the configuration policies which are bound to the cluster
+
+Each cluster's policies will be applied in the order defined by the ran.openshift.io/ztp-deploy-wave annotations.
+
 The progress of configuration policy reconciliation can be monitored in the ACM dash board.
 
+The final policy that will become compliant is the one defined in the `du-validator-policy-*` policies. This policy, when compliant on a cluster, corresponds to the ZTP process completing, ensuring that all cluster configuration, operator installation, and operator configuration has completed.
```

Comment:
Maybe mention here that TALO will set the ztp-done label on the cluster as well?

Reply:
Does it?  I didn't know if that merged yet...

Reply:
It hasn't yet. We should update when it does.